core/src/test/java/org/elasticsearch/search/aggregations/pipeline/BucketScriptIT.java

```diff
@@ -238,6 +243,52 @@ public void testInlineScript2() {
         }
     }
 
+    public void testInlineScript3() {
```

Comment:
can you give it a name that contains `dateHistogram` to make it clear what it tests that the other tests do not test